helper/configure-image.sh

```diff
@@ -24,26 +27,34 @@ else
     exit 1
 fi
 
-# Install package manager dependencies
+# Install dependencies using the determined package manager
 case "$PACKAGE_MANAGER" in
-        apt-get)
-            apt-get update
-            apt-get install -qq curl
-            ;;
-        apk)
-            apk add --update-cache --upgrade --virtual .temp-dde-deps curl shadow
+    apt-get)
+        apt-get update
+        apt-get install -qq curl
+        [ "$SHELL_TO_INSTALL" = "/bin/zsh" ] && apt-get install -y zsh
```

Comment:
@sbaerlocher this only works with zsh, bash should also be installed when chosen in the config file. We could just reuse $DDE_CONTAINER_SHELL

Reply:
@sbaerlocher why not directly use $DEFAULT_SHELL if set? Performance wise it would be better if we call apk add only once